src/routes/user/updateUser.ts

```diff
@@ -52,6 +63,7 @@ const editUserHandler = async (
       salt: hashObj !== undefined ? hashObj.salt : undefined,
       is_admin: isAdmin,
       is_blender: isBlender,
+      archived_at: archiveUser ? knexController.fn.now() : null,
```

Comment:
Nyt tää voi helposti flippaa takaisin arkistosta aktiiviseksi käyttäjäksi, pitäis varmaan tehdä asialle jotain